### PR TITLE
New layout

### PR DESCRIPTION
**Pull request by saraclay**

### CODE REVIEW

File: en-US/Docs/AdafruitMakerKit.md [hunk 1]
```diff
@@ -7,65 +7,33 @@ permalink: /en-US/Docs/AdafruitMakerKit.htm
...
   <div class="col-md-12 col-xs-24">
-    <center>
-      <h2 class="thin-header floatTop">Welcome! Let's get started.</h2>
-      <br/>
-      If you don't have one, you can order one <a target="_blank" href="http://adafruit.com/windows10iotpi2"> here.</a>
-    </center>
+    <h1>Windows 10 IoT Core Starter Pack</h1>
+    <p>Windows 10 IoT Core is a version of Windows targeted towards small embedded devices and maker boards. You can use Windows 10 IoT Core to read sensor data, control actuators, connect to the cloud, create IoT applications, and more.</p>
+    <p>With the parts included in the kit and the instructions provided here, you'll be ready to make your first project.</p>
+    <p><b>Not sure what this is about?</b><br>
```
Comment (bfjelds) [inline en-US/Docs/AdafruitMakerKit.md:16]: do we want to leave "Not sure what this is about?" in here?  is that a TODO message to us or is that what we're saying to the user?

Comment (saraclay) [reply en-US/Docs/AdafruitMakerKit.md:16]: That is to the user :)

Comment (bfjelds) [reply en-US/Docs/AdafruitMakerKit.md:16]: this just a nice transition to the Learn more line, isn't it?  if so, ignore my comment....just want to make sure it is on purpose.

Comment (saraclay) [reply en-US/Docs/AdafruitMakerKit.md:16]: Yes - but thank you for the detailed review 👍

**Pull request closed without merging**